### DISCUSSION TITLE
Respawning as a civilian and enableSaving

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1332] [Question and answer]
**Hello**,
I have a problem with the respawn: sometimes, when I die, I respawn as a civilian and I have to restart the mission to fix it. Also, I would like to ask where the parameter "enableSaving" is located since I would like to try the Vanilla saving system for the mission. Thanks in advance!

**Comment by Vdauphin** [comment]
Hello,

> I have a problem with the respawn: sometimes, when I die, I respawn as a civilian and I have to restart the mission to fix it. 

Don't change the mission file and you will not have problem

> Also, I would like to ask where the parameter "enableSaving" is located since I would like to try the Vanilla saving system for the mission.

It will not work. Use the one provided as it has better performance

Cheers

**Reply by Vsauce07** [reply]
Hi 

> Don't change the mission file and you will not have problem

I didn't change anything except for adding gear in the "chem" file.

> It will not work. Use the one provided as it has better performance

Problem is that I have a lot of problems playing the scenario using a dedicated server, starting from performance. Also, I play just with my brother and it's time-consuming every time starting the server.

**Reply by Vdauphin** [reply]
> I didn't change anything except for adding gear in the "chem" file.

Then it must be a mod you are loading, disable it

> Problem is that I have a lot of problems playing the scenario using a dedicated server, starting from performance. 

Well, with this approach client rendering and AI calculation wll be done on different core of your CPU. So it is better for performance

> Also, I play just with my brother and it's time-consuming every time starting the server.

You just have to double click on a .bat file to start it. Nothing time consuming here

Cheers

**Reply by Vsauce07** [reply]
> Then it must be a mod you are loading, disable it

I'm using KAT, maybe is that?

**Reply by Vdauphin** [reply]
I don't know
